src/Datadog.Trace/ServiceFabric/ServiceRemotingTags.cs

```diff
@@ -0,0 +1,61 @@
+#nullable enable
```

Comment:
I like this, we should start doing this for more of our classes 👍🏼

Reply:
Thanks! I started a branch a while back where I started doing this on core types like `Tracer`, `Span`, `Scope`: https://github.com/DataDog/dd-trace-dotnet/compare/lpimentel/nullable-references
It's only a few months behind `master` 😅 

At some point we should start enforcing this in new files going forward.